sdk/go/things.go

```diff
@@ -48,6 +52,60 @@ func (sdk mfSDK) CreateThing(thing Thing, token string) (string, error) {
 	return id, nil
 }
 
+func (sdk mfSDK) BulkCreateThings(path string, token string) ([]Thing, error) {
+
+	if _, err := os.Stat(path); os.IsNotExist(err) {
+		return []Thing{}, ErrInvalidArgs
+	}
+
+	things, err := thingsFromFile(path)
+	if err != nil {
+		return []Thing{}, err
+	}
+
+	data, err := json.Marshal(things)
+	if err != nil {
+		return []Thing{}, ErrInvalidArgs
+	}
+
+	endpoint := fmt.Sprintf("%s/%s", thingsEndpoint, "bulkCreate")
```

Comment:
Maybe this endpoint should just be `/things/bulk`. You already have `POST` method from which is clear that you're creating things.